servicenow.json

```diff
@@ -1048,6 +1035,24 @@
                     "data_path": "action_result.data.*.user_input",
                     "data_type": "string"
                 },
+                {
+                    "data_path": "action_result.data.*.vendor.display_value",
+                    "data_type": "string",
+                    "example_values": [
+                        "Microsoft"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.vendor.link",
+                    "data_type": "string",
+                    "example_values": [
+                        "https://dev78070.service-now.com/api/now/table/core_company/0e8b8e650a0a0b3b004f285ffbb1a4fc"
+                    ]
```

Comment:
Should we have "contains": "url" here as well?

Reply:
Yes, I have made the suggested changes.